app/components/experience.tsx

```diff
@@ -10,7 +11,12 @@ export function Experience() {
       <div data-theme-color="strawberry">
         <TicketContainer className="-rotate-2">
           <div className="flex flex-col font-semibold">
```

Comment:
Cleaned these up to allow graceful line breaks (and disappearing dashes). This should be its own component, something like IndexCard